westcoast/src/main/java/frc/robot/Robot.java

```diff
@@ -47,7 +55,7 @@ public void autonomousPeriodic() {
 
   @Override
   public void teleopInit() {
-
+    teleop.schedule();
```

Comment:
```suggestion
    teleop.start();
```
I think that is the proper way to start a command.